### ISSUE TITLE
Ecommerce plan: My plan page does not mention store creation

### ISSUE THREAD

**Issue by alaczek** [Automattic/wp-calypso#33010]
When signing up and purchasing Ecommerce plan, clicking the "Confirm Now" button in the email takes customer to My plan page in Calypso (https://wordpress.com/plans/my-plan/{site}/?verified=1):

![image](https://user-images.githubusercontent.com/4550351/57671919-1d76c280-7659-11e9-8125-c2536363c69a.png)

The features list does not mention ecommerce/starting a store, which presumably is the main reason someone would purchase Ecommerce plan. Ideally the whole flow would get an overhaul and we'd guide customers through store setup as part of onboarding, but even adding a card with a nudge towards setting up a store to the top of features list would be an improvement.

The whole video of signup process can be found in this post (under eCommerce plan customers): paObgF-7Y-p2

**Comment by jeffikus** [comment]
I agree, this should direct people to the setup for the eCommerce plan. Perhaps adding a Call to Action in the "Your site is on a eCommerce plan" hero area?

**Comment by jeffikus** [comment]
Here is an example of a call to action to setup a store: https://github.com/Automattic/wp-calypso/issues/33397

**Comment by jeffikus** [comment]
## Design Options

### Existing components

I've tinkered with some design options if we wanted to add a call to action for this. I've experimented with adding a card below the "My Plan: eCommerce" header card based on one of the cards from the stats screen which gets customers to Start the Google Business setup, see 
![wordpress com_stats_day_wootothecommercecalypsotest wpcomstaging com](https://user-images.githubusercontent.com/937325/60185061-8a20d780-9829-11e9-912b-42962c121584.png)

I also used the same welcome image from the checklist page at wp-admin/admin.php?page=wc-setup-checklist see: 
![wootothecommercecalypsotest wpcomstaging com_wp-admin_admin php_page=wc-setup-checklist](https://user-images.githubusercontent.com/937325/60185393-26e37500-982a-11e9-8e14-5049afc196a8.png)
 
### Card Design Options
Option 1:
![wordpress com_plans_my-plan_wootothecommercecalypsotest wpcomstaging com__verified=1 (4)](https://user-images.githubusercontent.com/937325/60185112-a6247900-9829-11e9-8e11-6d3cbab3361e.png)

Option 2:
![wordpress com_plans_my-plan_wootothecommercecalypsotest wpcomstaging com__verified=1 (3)](https://user-images.githubusercontent.com/937325/60185132-af154a80-9829-11e9-9aa3-44494b31f44c.png)

After adding to the page it looks like:
![wordpress com_plans_my-plan_wootothecommercecalypsotest wpcomstaging com__verified=1 (6)](https://user-images.githubusercontent.com/937325/60185181-c3594780-9829-11e9-9415-199bef0bf5f3.png)

With all elements in full page view:
![wordpress com_plans_my-plan_wootothecommercecalypsotest wpcomstaging com__verified=1 (5)](https://user-images.githubusercontent.com/937325/60185314-f69bd680-9829-11e9-96df-b1007779bf6b.png)

**Comment by sixhours** [comment]
@jeffikus I like this! I think it makes sense as a call-to-action when someone has first started their site.

What would this look like when someone has already set up the store? I feel like there should be a card to represent the store under "eCommerce Plan Features".

**Comment by davipontesblog** [comment]
Moved to WordPress.com Ecommerce Plan project for triaging.

**Comment by Aurorum** [comment]
I wonder if this would now be best suited as a Task part of Customer Home?
